src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java

```diff
@@ -114,6 +118,7 @@ private String buildVendorAddressSql(JaggaerContractUploadProcessingMode process
         sb.append(buildJoinClauseWithPOVendors(true));
         sb.append(buildRestrictActiveRows(true));
         sb.append(buildTimeFrameRestrictionClause(processingMode, processingDate));
+        sb.append("AND VA.VNDR_ADDR_TYP_CD in ('RM', 'PO') ");
```

Comment:
You may want to create a follow-up user story to make the SQL building a little cleaner and to create it as a parameterized query. Some previous PR(s) introduced some helper classes (like CuSqlQuery and CuSqlChunk) that make it simpler to build a parameterized SQL query.

Reply:
I am not sure if making this query parameterized is ideal.  I like being able to log the SQL being executed to help debug any problems. Parameterizing SQL statements is a nice way of preventing user input SQL injection.  We don't have user input here.
If you think there is an alternative way of creating this SQL that is simpler, please let me know and I can create a follow up user story to adjust the SQL later.

Reply:
For the example line given here, it's okay to use the hard-coded SQL literals for now, though ideally it would be preferable to use constants to store the values and then pass them in as query parameters. However, I've noticed other areas in this DAO class where the code is appending a String-formatted date into the SQL, and is using a SQL TO_DATE function to convert the String back into a Date on the database side. At some point in the future, it would be better to pass such values into the query as DATE or TIMESTAMP JDBC parameters.

The CuSqlQuery and CuSqlChunk helper classes I mentioned earlier could help simplify building this query in a future user story. They make it more convenient to set up parameterized queries, and they also keep track of the query parameters so that both the query string and the parameters can easily be written to the logs as needed. The classes CuSqlQueryTest and DocumentMaintenanceDaoJbc have examples of the capabilities of these helper objects.

Reply:
I created https://jira.cornell.edu/browse/KFSPTS-26048